test-plone-5.1.x.cfg

```diff
@@ -0,0 +1,13 @@
+[buildout]
+extends =
+    https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/test-plone-5.1.x.cfg
+    sources.cfg
+
+package-name = ftw.faqblock
+
+versions = versions
+
+
+[versions]
+setuptools = <45a
```

Comment:
I know we did that in some pkgs, but in my opinion that should go to ftw-buildouts if possible.

Reply:
We need this because v45 and later is not py2 compatible.

Reply:
I know, but that doesn't mean that we have to do it here. All our packages depend on that being pinned. Why not put it into ftw.buildouts?